src/store/apis/index.jsx

```diff
@@ -0,0 +1,3 @@
+import { useSearchHotelsQuery} from './hotels/searchHotels.endpoint';
```

Comment:
This file should contains list of exports. There is an example for `searchHotels.endpoint`
```suggestion
export * from './hotels/searchHotels.endpoint';
```
Then you can make the following import shorter:
`import { useSearchHotelsQuery } from "../../store/apis/hotels/searchHotels.endpoint";` -> `import { useSearchHotelsQuery } from '../../store/apis';`

Reply:
![cr](https://github.com/user-attachments/assets/d7f1d0f1-a584-46ff-a4df-20234ab35beb)
I think import is not needed here because it causes an error